client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/RequestPermissionBottomSheet.kt

```diff
@@ -0,0 +1,76 @@
+package presentation.login.composable
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import com.beepbeep.designSystem.ui.composable.BpButton
+import com.beepbeep.designSystem.ui.composable.BpExpandableTextField
+import com.beepbeep.designSystem.ui.composable.BpTextField
+import com.beepbeep.designSystem.ui.composable.BpTransparentButton
+import com.beepbeep.designSystem.ui.theme.Theme
+import presentation.login.LoginScreenInteractionListener
+import presentation.login.LoginScreenUIState
+import resources.Resources
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RequestPermissionBottomSheet(
+    listener: LoginScreenInteractionListener,
+    state: LoginScreenUIState,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.wrapContentHeight()
+            .padding(horizontal = Theme.dimens.space16, vertical = Theme.dimens.space24)
+    ) {
+        Text(
+            text = Resources.strings.askForPermission,
+            color = Theme.colors.contentPrimary,
+            style = Theme.typography.titleLarge,
+        )
+        BpTextField(
+            text = state.restaurantName,
+            label = Resources.strings.restaurantName,
+            keyboardType = KeyboardType.Text,
+            onValueChange = listener::onRestaurantNameChanged,
+            modifier = Modifier.fillMaxWidth().padding(top = Theme.dimens.space24)
+        )
+        BpTextField(
+            text = state.ownerEmail,
+            onValueChange = listener::onOwnerEmailChanged,
+            label = Resources.strings.userEmailLabel,
+            keyboardType = KeyboardType.Text,
+            modifier = Modifier.fillMaxWidth().padding(top = Theme.dimens.space16),
+        )
+        BpExpandableTextField(
+            text = state.description,
+            onValueChange = listener::onDescriptionChanged,
+            label = Resources.strings.whyBeepBeep,
+            hint = Resources.strings.questionHint,
+            keyboardType = KeyboardType.Text,
```

Comment:
I think there is no need to use `keyboardType `when it is by default is 
```kotlin
KeyboardType.Text
```

Reply:
thanks for your review